RevenueCatUI/CustomerCenter/Views/PromotionalOfferView.swift

```diff
@@ -97,18 +102,28 @@ struct PromotionalOfferHeaderView: View {
     @ObservedObject
     private(set) var viewModel: PromotionalOfferViewModel
 
+    private enum Constants {
+        static let spacing: CGFloat = 30
+        static let topPadding: CGFloat = 150
+        static let horizontalPadding: CGFloat = 40
+    }
```

Comment:
Not a blocker at all, but I'd be fine with these being directly in the `PromotionalOfferHeaderView`, outside of the enum. (Unless there's a reason to keep them in the enum of course.)

(This applies to the other `Constants` enum in the `PromotionalOfferView` too.)